language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/resolvers/parsercontext/ParserRuleMatchStatementContextResolver.java

```diff
@@ -95,10 +95,13 @@
                     completionItems.add(this.fillInvokableSymbolMatchSnippet((BInvokableSymbol) bSymbol, ctx));
                 } else if (!(symbolInfo.getScopeEntry().symbol instanceof BInvokableSymbol)
                         && bSymbol instanceof BVarSymbol) {
-                    this.fillVarSymbolMatchSnippet((BVarSymbol) bSymbol, completionItems, ctx);
-                    completionItems.add(this.populateVariableDefCompletionItem(symbolInfo));
+                    fillVarSymbolMatchSnippet((BVarSymbol) bSymbol, completionItems, ctx);
```

Comment:
For better readability, let's use the `this` when calling methods in the same instance

Reply:
It looks redundant to me. [SO answers](https://stackoverflow.com/questions/14932697/java-what-is-considered-more-readable-this-or-no-this) confirms the same. And there's no performance gain either(java compiler produces same bytecode for both). We can use `this` in ambiguous states as below(even we can simply rename the variable);
```
public MyClass(String data) {
    this.data = data
```
WDYT?